elsie/boxmixin.py

```diff
@@ -172,8 +196,25 @@ def rect(
         stroke_dasharray=None,
         rx=None,
         ry=None,
-    ):
-        """ Draw a rect around the box """
+    ) -> "BoxMixin":
```

Comment:
I would put here BoxItem, not BoxMixin. It helps to distinguish when a new box is created and when BoxItem is created